llama_stack/providers/inline/agents/meta_reference/agent_instance.py

```diff
@@ -1077,6 +1077,8 @@ async def execute_tool_call_maybe(
     tool_to_group: Dict[str, str],
 ) -> ToolInvocationResult:
     name = tool_call.tool_name
+    if not name:
```

Comment:
can you show the _server_ side stack trace which led to the error? Your PR description has client-side trace.

Reply:
@hemajv Could you provide this info?